hack/lib/util/environment.sh

```diff
@@ -253,6 +253,13 @@ function os::util::environment::setup_server_vars() {
     NODE_CONFIG_DIR="${SERVER_CONFIG_DIR}/node-${KUBELET_HOST}"
     export NODE_CONFIG_DIR
 
+    ETCD_CLIENT_CERT="${MASTER_CONFIG_DIR}/master.etcd-client.crt"
```

Comment:
Why?

Reply:
Because I can guarantee that MASTER_CONFIG_DIR will be set here. Also, these are needed for cleanup